src/GafferUI/TextGadget.cpp

```diff
@@ -90,7 +99,15 @@ void TextGadget::renderLayer( Layer layer, const Style *style, RenderReason reas
 		return;
 	}
 
-	style->renderText( Style::LabelText, m_text );
+	if( m_dimmed )
+	{
+		Imath::Color4f dimColor( 0.75, 0.75, 0.75, 1.0f );
+		style->renderText( Style::LabelText, m_text, Style::NormalState, &dimColor );
+	}
+	else
+	{
+		style->renderText( Style::LabelText, m_text );
+	}
```

Comment:
Any reason we shouldn't let style make these decisions by passing `Style::DisabledState` like we're doing for the nodes and connections?

Reply:
Addressed in f7d5ecf9